components/Hero.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <!-- hero -->
+      <!-- hero wrapper -->
+      <div
+        class="hero-wrapper grid grid-cols-1 md:grid-cols-12 gap-8 items-center"
+      >
+        <!-- hero text -->
+        <div class="hero-text col-span-6">
+          <h1
+            class=" font-bold text-4xl md:text-5xl max-w-xl text-gray-900 leading-tight"
+          >
+            Making Open Source Look Different
+          </h1>
+          <hr class=" w-12 h-1 bg-nebula rounded-full mt-8" />
+          <p class="text-gray-800 text-base leading-relaxed mt-8 font-semibold">
+            We provide an open platform wer any developer can share, contribute
+            or donate on any open source project!
```

Comment:
```suggestion
            or donate on any open source projects!
```
Typo

Reply:
Thanks 👍 .Will be taken into account in my next commit.